lib/core/configurator.c

```diff
@@ -688,6 +688,17 @@ static int on_config_mime_setdefaulttype(h2o_configurator_command_t *cmd, h2o_co
     return 0;
 }
 
+static const char *get_ext(h2o_configurator_command_t *cmd, yoml_t *node)
+{
+    if (strcmp(node->data.scalar, "default") == 0) {
+        return node->data.scalar;
+    } else if  (assert_is_extension(cmd, node) == 0) {
+        return node->data.scalar + 1;
```

Comment:
The function seems to either return "default" or return the extension stripping the preceding dot.

Does the fact mean that we can no longer register `.default` as an extension?